### PR TITLE
style: Fix repeated-append (FURB113)

### PR DESCRIPTION
**Pull request by echoix**
Ruff rule: https://docs.astral.sh/ruff/rules/repeated-append/

### CODE REVIEW

**Comment by echoix** [comment]
I'm starting to get too much rebase conflicts, so the rest of the queued changes for PRs will be once the currently open PRs are merged, so somewhere in the next couple days, when I have some computer time available.

File: utils/mkhtml.py [hunk 1]
```diff
@@ -418,11 +418,9 @@ def get_last_git_commit(src_dir, addon_path, is_addon):
             return get_git_commit_from_file(src_dir=src_dir)
 
 
-html_page_footer_pages_path = (
-    os.getenv("HTML_PAGE_FOOTER_PAGES_PATH")
+html_page_footer_pages_path = os.getenv("HTML_PAGE_FOOTER_PAGES_PATH")
```
Comment (wenzeslaus) [inline utils/mkhtml.py:421]: This is the same type of unrelated fix as before. ...and several issues are reported here (which I guess is a good thing).

**Review by wenzeslaus — CHANGES_REQUESTED** [review]
Sometimes the original append code is easier to read and maintain than the extent one-liner, but overall this is a good change.

I'm wondering how the tuples are viewed here. I remember that some later practices were pushing for lists as semantically better than tuples which were originally used for performance reasons in this context.

File: python/grass/pygrass/vector/basic.py [hunk 2]
```diff
@@ -133,9 +133,7 @@ def contains(self, point):
 
         """
         return bool(
-            libvect.Vect_point_in_box(
-                point.x, point.y, point.z if point.z else 0, self.c_bbox
-            )
+            libvect.Vect_point_in_box(point.x, point.y, point.z if point.z else 0, self.c_bbox)
```
[the remark this answers is not in the document]
Comment (echoix) [reply python/grass/pygrass/vector/basic.py:136]: I’ll take a look and compare to my source branch. I have a big branch where I set up Pylint 3.x, and I fix issue after issue to reduce the diff of what I need to configure Pylint to ignore. Since having PRs merged is usually way too long here, I have them all in my branch, and then I cherry pick one commit to a new branch taken from main.

So the unrelated change might just be the fact that it isn’t in the same order as it was before being cherry picked, and I didn’t spot it when creating that branch.

I know that the combined changes are correct.

File: utils/mkhtml.py (hunk 1 above)
Comment (echoix) [reply utils/mkhtml.py]: done

File: python/grass/pygrass/vector/basic.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (echoix) [reply python/grass/pygrass/vector/basic.py]: done

**Comment by echoix** [comment]
The large part of weird changes here were conflicts because of a bad rebase with https://github.com/OSGeo/grass/pull/4013 that was before this commit in the source.

**Comment by echoix** [comment]
> Sometimes the original append code is easier to read and maintain than the extent one-liner, but overall this is a good change.
> 
> 
> 
> I'm wondering how the tuples are viewed here. I remember that some later practices were pushing for lists as semantically better than tuples which were originally used for performance reasons in this context.

What would you want here? Was there a real problem or not yet? Do you request changes on something?

**Comment by echoix** [comment]
I plan on spending a bit of time this evening on this and the other PR that had a comment. @wenzeslaus Would you mind taking a new look to keep this PR series going?

(I don't want to get bored without finishing having Pylint 3.x working)

**Review by wenzeslaus — APPROVED** [review]
